app/controllers/v0/vic/profile_photo_attachments_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module V0
+  module VIC
+    class ProfilePhotoAttachmentsController < ApplicationController
+      include FormAttachmentCreate
+
+      def create
+        form_attachment = ::VIC::ProfilePhotoAttachment.new
+
+        form_attachment.set_file_data!(
+          params[:profile_photo_attachment][:file_data],
+          get_in_progress_form
+        )
+        form_attachment.save!
+        render(json: form_attachment)
+      end
+
+      private
+
+      def get_in_progress_form
+        return nil if @current_user.blank?
+
+        form = InProgressForm.where(form_id: 'VIC', user_uuid: @current_user.uuid)
+                             .first_or_initialize(form_id: 'VIC',
```

Comment:
just doing `first_or_initialize` should fill in form_id and user_uuid. i would leave the rest of the fields alone

Reply:
I had it that way initially, but it threw an error when I tried to save saying form_data must not be empty.

Reply:
Nevermind! I must have been setting a value incorrectly. Fixing this.